.github/workflows/check-external-links.yaml

```diff
@@ -0,0 +1,40 @@
+name: Check external links
+
+on: workflow_dispatch
+
+jobs:
+  check-links:
+    name: Check for broken external links
+    runs-on: 'ubuntu-latest'
+
+    strategy:
+      fail-fast: false
```

Comment:
**issue** Given this workflow is a series of actions (rather than multiple actions running in parallel, like we do with [the different linting using a `matrix` of commands](https://github.com/alphagov/govuk-design-system/blob/main/.github/workflows/test.yaml#L64-L88)), we can happily [let it stop at the first error it encounters](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#handling-failures) (as there's no matrix whose jobs we may still want to keep running). 

```suggestion
```